packages/multichain-transactions-controller/CHANGELOG.md

```diff
@@ -7,11 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.0]
```

Comment:
I did not make it a major since this controller is still quite under heavy development, but we could bump the major still...

Any other opinions on this one?